src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -364,9 +364,9 @@ public function testOverwriteMultipleProgressBarsWithSectionOutputs()
         $this->assertEquals(
             '  0/50 [>---------------------------]   0%'.\PHP_EOL.
             '  0/50 [>---------------------------]   0%'.\PHP_EOL.
-            "\x1b[1A\x1b[0J".'  1/50 [>---------------------------]   2%'.\PHP_EOL.
```

Comment:
Again, the new test appears t make sense. 4 lines precede the line containing the clear character. The 4 lines are comprised of 2 PHP_EOLs and 2 strings.